lib/datasource/remote/model/invite_model.dart

```diff
@@ -1,24 +1,29 @@
 import 'package:equatable/equatable.dart';
+import 'package:seeds/utils/string_extension.dart';
 
 class InviteModel extends Equatable {
-  final int? inviteId;
-  final String? transferQuantity;
-  final String? sowQuantity;
-  final String? sponsor;
+  final int inviteId;
+  final String transferQuantity;
+  final String sowQuantity;
+  final String sponsor;
   final String? account;
-  final String? inviteHash;
-  final String? inviteSecret;
+  final String inviteHash;
+  final String inviteSecret;
```

Comment:
Nice is good to know most of this fields already have a default empty value.

Reply:
The HTTP calls to the blockchain always return "" - empirical observation

I believe this is because all values are always initialized in on chain tables - there's a default for each field.

Reply:
it would be to review all of the serialization models, these defaults values would save a lot of nullable fields